lib/markdown2.py

```diff
@@ -1738,7 +1740,7 @@ def unhash_code(codeblock):
                     codeblock = codeblock.replace(old, new)
                 return codeblock
             lexer = self._get_pygments_lexer(lexer_name)
-            if lexer:
+            if lexer and self.extras.get('no-code-highlighting', True) is True:
                 codeblock = unhash_code( codeblock )
```

Comment:
Oh yes, I'm stupid... Would you mind if I add `.keys()`, so we understand that it's a dict? I do this all the time, and it's way easier for me to read...